tests/utils/extended_app/migrations/0001_initial.py

```diff
@@ -1,6 +1,4 @@
-# -*- coding: utf-8 -*-
 # Generated by Django 1.9.5 on 2016-04-20 13:41
```

Comment:
we'd better drop migrations for this app and create a new one to sync auto fields changes in 3.1